Compiler/src/ssair/irinterp.jl

```diff
@@ -42,7 +42,7 @@ function abstract_eval_invoke_inst(interp::AbstractInterpreter, inst::Instructio
     else
         code = ci::CodeInstance
     end
-    argtypes = collect_argtypes(interp, stmt.args[2:end], StatementState(nothing, false), irsv)
+    argtypes = collect_argtypes(interp, stmt.args[2:end::Int], StatementState(nothing, false), irsv)
```

Comment:
I don't understand how `end` has to be type asserted... In any case this looks really weird so might need a comment or some kind of small refactor?

Reply:
`end` is `lastindex(stmt.args)` in this case

Reply:
Are we missing an assert that `stmt::Expr` above?